pkg/urfave/flags.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2019, the Drone Plugins project authors.
+// Please see the AUTHORS file for details. All rights reserved.
+// Use of this source code is governed by an Apache 2.0 license that can be
+// found in the LICENSE file.
+
+package urfave
+
+import (
+	"strings"
+
+	"github.com/urfave/cli"
+)
+
+// ShowAllFlags sets Hidden to false on all of the flags.
+//
+// By default all Drone environment flags are hidden by default. For
+// notification plugins its unlikely this is the desired behavior. This function
+// modifies all the flags to be shown in the help menu.
+func ShowAllFlags(flags []cli.Flag) []cli.Flag {
+	for i := 0; i < len(flags); i++ {
+		flags[i] = ShowFlag(flags[i])
+	}
+
+	return flags
+}
+
+// ShowFlagsByName sets Hidden to false on all flags with the given names.
+//
+//     flags := urfave.ShowFlagsByName(flags, urfave.RepoNameFlag, urfave.RepoOwnerFlag)
+func ShowFlagsByName(flags []cli.Flag, names ...string) []cli.Flag {
```

Comment:
Ok, with this kind of selector the constants from https://github.com/drone-plugins/drone-plugin-lib/issues/5#issuecomment-541511347 make sense again...